projects/knora-ui/src/lib/viewer/operations/display-edit/display-edit.component.scss

```diff
@@ -0,0 +1,3 @@
+::ng-deep .parent-value-component .mat-form-field {
```

Comment:
for your information: https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Reply:
😣technology evolves too fast. I'll update my code. Thanks!

Reply:
> ::ng-deep, /deep/ and >>> deprecation
> The ::ng-deep pseudo-class selector also has a couple of aliases: >>> and /deep/, and all three are soon to be removed.
> 
> The main reason for that is that this mechanism for piercing the style isolation sandbox around a component can potentially encourage bad styling practices.
> 
> The situation is still evolving, but right now, ::ng-deep can be used if needed for certain use cases.

https://blog.angular-university.io/angular-host-context/

From what I can find online, Angular has marked the ::ng-deep selector as deprecated but has not yet provided a new solution. They recommend to use ::ng-deep only if absolutely necessary so I shall use it sparingly.